api/documentation/controller.py

```diff
@@ -405,6 +414,50 @@ def addSchemes(self):
             }
         )
 
+        self.addComponent(
+            'schema', 'MARCRollContributorsDict', 'dict',
+            {
+                'items': {
+                    'type': 'object',
+                    'properties': {
+                        'MARCRoleCode': {'type': 'string'},
+                        'ContributorRole': {'type': 'string'},
+                        'MARCRoleCode': 'ContributorRole'
+                    }
+                }
+            }
+        )
+        self.addComponent(
+            'schema', 'LanguageCodes', 'dict',
+            {
+                'items': {
+                      'type': 'object',
+                      'properties': {
+                          'language_code': {'type': 'string'},
+                          'languages': {'type': "List['string']"},
+                          'language_code': 'languages'
```

Comment:
Same comment as above. And if this should be an enumerated list (for the purposes of this documentation I don't think it needs to be) that should be it's own component!